src/installer.ts

```diff
@@ -366,6 +439,17 @@ function evaluateVersions(versions: string[], versionSpec: string): string {
   return version;
 }
 
+export function getNodejsDistUrl(version: string) {
+  const prerelease = semver.prerelease(version);
+  if (version.includes('nightly')) {
+    return 'https://nodejs.org/download/nightly';
+  } else if (prerelease) {
```

Comment:
Unnecessary else statement
```suggestion
  } 
  if (prerelease) {
```